index.html

```diff
@@ -58,8 +58,8 @@ <h2>Conrad’s Creative Mode 2.0</h2>
 
       <p><b>Contributions</b></p>
       <ul>
-        <li>Reduced CPU utilization by 42% through refactoring the code base to reduce the time complexity of several sections of code.</li>
-        <li>Improved readability of code by introducing abstraction through the creation of over 250 functions to encapsulate various portions of the code.</li>
+        <li>42% reduction in CPU utilization through code refactoring to reduce time complexity of several sections of code.</li>
```

Comment:
These small edits are good to go without approval from someone other than the pusher. Also, the pusher is the owner (me) and the sole person working on this project.